gouttelette/cmd/refresh_modules.py

```diff
@@ -1115,7 +1115,18 @@ def init_resources(paths: str) -> Dict:
 def generate_amazon_cloud(args: Iterable):
     module_list = []
 
-    for type_name in RESOURCES:
+    if args.get("resource"):
+        modules_file_path = pathlib.Path(args.get("resource") + "/modules.yaml")
+    else:
```

Comment:
args["resource"] will never be empty. It will either take the user input or take the default value before reaching this point.  So this if else condition is not needed. Refer : https://github.com/ansible-collections/gouttelette/blob/1e6d6293bfe2b07a5a4d83ddd91566b188c6dcc6/gouttelette/cmd/refresh_modules.py#L1384